rdbi-recipes/src/test/resources/suites/standard.xml

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+<suite name="UnitTests">
+    <test name="standard">
+        <groups>
+            <run>
+                <exclude name="integration" />
+            </run>
+        </groups>
+        <packages>
+            <package name="com.lithium.dbi.rdbi.recipes.cache" />
+            <package name="com.lithium.dbi.rdbi.recipes.channel" />
+            <package name="com.lithium.dbi.rdbi.recipes.locking" />
+            <package name="com.lithium.dbi.rdbi.recipes.presence" />
+            <package name="com.lithium.dbi.rdbi.recipes.scheduler" />
+            <package name="com.lithium.dbi.rdbi.recipes.set" />
+            <package name="com.lithium.dbi.rdbi.ratelimiter" />
+            <package name="com.lithium.dbi.rdbi" />
+        </packages>
+    </test>
```

Comment:
any particular reason you expanded the wildcard to list individual package names here and in integration.xml ?